thrift/src/main/thrift/atoms/recipe.thrift

```diff
@@ -32,25 +32,37 @@ struct Range {
 struct Ingredient {
   1: required string item
   2: optional string comment
-  3: optional double quantity
+  // 3: optional double quantity
   4: optional Range quantityRange
   5: optional string unit
+  6: optional string dietaryInfo
+  7: optional Range caloryRange
+  8: optional Range quantityRange
```

Comment:
Is `8:` a duplicate of `4:` (`quantityRange`)?

Reply:
Ah shoot, my bad. Fixed!